nncf/tensorflow/accuracy_aware_training/runner.py

```diff
@@ -83,22 +90,55 @@ def dump_checkpoint(self, model):
         model.save_weights(checkpoint_path)
 
         if self.best_val_metric_value == self.current_val_metric_value:
-            best_path = osp.join(self._checkpoint_save_dir,
-                                 'acc_aware_checkpoint_best_compression_rate_'
-                                 '{comp_rate:.3f}.pth'.format(comp_rate=self.compression_rate_target))
-            self._best_checkpoints[self.compression_rate_target] = best_path
+            best_checkpoint_filename = 'acc_aware_checkpoint_best.pth'
```

Comment:
Seems like it is applicable only for PyTorch.

Reply:
@lzrvch please take a look

Reply:
Yeah, checkpoints are saved in TF format anyway, but please rename to `.ckpt` to avoid confusion.

Reply:
Done